src/medali/lib/harmonic_params/V1M1.ini

```diff
@@ -0,0 +1,98 @@
+# Harmonic parameters metadata, version 1.0
+
+[Metadata]
+
+# creation date - date of the data processing
+date_creation: datetime
+
+# modification date - date of last modification, by default set the same as the creation time when processing the data
+date_modification: datetime
+
+# dataset creator
+creator: string
+
+# name of the used worker software
+worker_name: string
+
+# tag of the used worker software
+worker_git_tag: string
+
+# 7-digit commit of the used worker software
+worker_git_commit: string
+
+# name of the used wrapper software
+wrapper_name: string
+
+# tag of the used wrapper software
+wrapper_git_tag: string
+
+# 7-digit commit of the used wrapper software
+wrapper_git_commit: string
+
+# input data version version of the input data used for harmonic regression computation (e.g. V01R01)
+input_data_version: string
+
+# variable name - name of the variable used for harmonic regression computation ( e.g. sig0, gam0, plia, ...)
+input_variable_name: string
+
+# input files number - number of input files
+input_files_number: integer
+
+# relative orbit number
+orbit_relative: integer
+
+# orbit direction (ascending or descending)
+orbit_direction: string
+
+# acquisition mode id
+mode: string
+
+# band of the image (e.g. polarisation or name of the respective band)
+band: string
+
+# tile id - EQUI7 tile ID
+tile_id: string
+
+# run number
+run_number: integer
+
+# harmonic parameter - name of the respective harmonic parameter
+harmonic_parameter: string
+
+# start date - starting date of the harmonic regression computation
+start_date: datetime
+
+# end date - end date of the harmonic regression computation
+end_date: datetime
+
+# k value - k value of the harmocnic regression
+k_value: integer
+
+# redundancy of the least-squares adjustment
+redundancy: integer
+
+# number of obervation available - information if the number of observation layer is available
+num_obs_available: boolean
+
+# verbal description how the values were encoded
+encoding_info: string
+
+# verbal description of the values
+value_info: string
+
+[Expected_value]
+
+# acquisition mode id
+mode: list, IW, EW
+
+# orbit direction id
+orbit_direction: list, A, D
+
+# band
+band: list, VH, VV
+
+# variable name id
+variable_name: list, SIG0, GMR0
```

Comment:
'GMR0' does not exist and is called 'GMR' in ACube. If you plan to use it please keep it , otherwise I think we can remove it for the time being.